app/pb_kits/playbook/pb_typeahead/_typeahead.scss

```diff
@@ -0,0 +1,52 @@
+.pb_typeahead {
+  .pb_typeahead_wrapper {
+    position: relative;
+  }
+
+  .text_input_wrapper {
+    margin-bottom: 0;
+  }
+
+  .pb_item_kit {
+    padding: 10px 0;
+
+    &:hover {
+      background-color: $bg_light;
+    }
+  }
+
+  &:focus-within [class^=pb_list_kit] {
+    max-height: 18em;
+    overflow-y: auto;
+    overscroll-behavior: contain;
+    position: absolute;
+    left: 0;
+    right: 0;
+    background: white;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
```

Comment:
This should be leveraging shadow values that have already been UX approved `app/pb_kits/playbook/tokens/_shadows.scss`

Reply:
I feel like this could also be a results kit...but we could abstract in a later release

Reply:
@jasoncypret I agree, I am sure we could extract out more than just results as a new kit. I like the idea of punting that for now.

Reply:
@thestephenmarshall Any suggestions for the shadows?

Reply:
@terryfinn try `box-shadow: $shadow_deep;` first